modules/kubeflow-mlflow/tests/test_deployment.py

```diff
@@ -1,4 +1,5 @@
 import subprocess
+from email.policy import default
```

Comment:
question: I'm not sure where this import is used
```suggestion
from email.policy import default
```

Reply:
Uhm, interesting. I'm not really sure how this got it actually. I'll remove this